src/pages/AddItem.js

```diff
@@ -52,13 +52,19 @@ export default function AddItem({ token }) {
     const itemExists = doesItemExistInDatabase(itemName);
 
     if (itemExists) {
-      swal(
-        'OH GOSH!',
-        `${normalizeString(itemName).toUpperCase()} is already in your list`,
-        'error',
-      );
+      Swal.fire({
+        title: 'OH GOSH!',
+        text: `${normalizeString(
+          itemName,
+        ).toUpperCase()} is already in your list`,
+        icon: 'error',
+      });
```

Comment:
I see what you mean @zeeatwork about the improved format for structuring the message from SWA1 to SWA2